### ISSUE TITLE
Cannot proceed past "parity signer new-token" pop-up window

### ISSUE THREAD

**Issue by jfnz** [paritytech/parity#7745]
> I'm running: Ubuntu 16.04.3 LTS (GNU/Linux 4.4.0-112-generic x86_64)
>
> - **Which Parity version?**: Parity/v1.9.0-beta-53ec114-20180125/x86_64-linux-gnu/rustc1.23.0
> - **Which operating system?**: Linux
> - **How installed?**: bash <(curl https://get.parity.io -kL)
> - **Are you fully synchronized?**: no
> - **Which network are you connected to?**: ethereum
> - **Did you try to restart the node?**: yes

Fresh install of Parity on Ubuntu server. When I log in to the server I'm asked to generate a new token using "parity signer new-token" however once I enter the token, or visit the token URL (e.g. http://10.11.12.20:8180/#/auth?token=xxxxxxxxxxx) nothing happens - it gets stuck with that pop-up window in the browser and I can see some page content in the background but I can't click out of the pop-up.

![image](https://user-images.githubusercontent.com/12504630/35590255-2428d366-066b-11e8-9d6f-dc5a726913a6.png)

This happens regardless of whether the page is served via HTTP or HTTPS and I've tried using incognito mode to make sure it wasn't an extension or similar. Webserver is nginx configured as per instructions here: https://paritytech.github.io/wiki/Wallet-Remote-Access

**Comment by 5chdn** [comment]
I would recommend downgrading to 1.8.7 and consider running a `--public-node` wallet.

Debugging the wallet remote access is a pain in some cases. If you want to go down that rabbit hole: Please, share your exact node configuration :)

**Comment by jfnz** [comment]
Thanks -- how do I specify which version I want to install if I'm using the "one-click" installer? I don't really want to compile from source and worry about dependencies, etc.

**Comment by 5chdn** [comment]
bash <(curl https://get.parity.io -kL) -r stable

will get you the latest stable release (currently 1.8.7)

**Comment by jfnz** [comment]
Okay, thank you. Is there any way I can remove the existing version given it wasn't installed with a package manager?

I have a snapshot of the VM, but have made additional changes since I'd prefer not to lose by reverting.

Thanks for your help - I sincerely appreciate it 😁

**Comment by 5chdn** [comment]
sudo apt purge parity

should do the trick

**Comment by jfnz** [comment]
```
media@ubuntu:~$ sudo apt purge parity
[sudo] password for media:
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package parity
```

**Comment by jfnz** [comment]
Okay, so I just removed the parity folder and the parity binary from /usr/bin. I have now installed the stable release of Parity (v1.8.9-stable-98cccd5-20180201/x86_64-linux-gnu/rustc1.23.0).

I get the same exact result, can't get rid of this pop-up whether I paste the token in the box or I use the token URL.

![image](https://user-images.githubusercontent.com/12504630/35773292-3079d724-09b2-11e8-9b03-7ec9712a0253.png)


What is going on? 0_o
